<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LiveScore Futbol</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Recursos/iconos/ico.png</normaloff>../Recursos/iconos/ico.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tbCompeticion">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>País</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Competicion</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Traduccion</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbJuegos">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Hora</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tiempo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Equipo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Resultado</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Equipo</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDescargar"/>
   <addaction name="actionCrear_Archivo"/>
   <addaction name="actionSalvar_Comp"/>
   <addaction name="actionConfigurar"/>
   <addaction name="separator"/>
   <addaction name="actionActualizar"/>
   <addaction name="actionDetener"/>
   <addaction name="actionAyuda"/>
  </widget>
  <widget class="QDockWidget" name="dockWConfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Configuración</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabConfGeneral">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>URLs de Descarga</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>index</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leditUrlIndex"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Banderas</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="leditUrlFlags"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Directorios</string>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="6" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Salvar Datos</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Recursos</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="leditDirRecursos"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="leditDirFichConf"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="leditDirSalvaDatos"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Fichero Conf.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Actualizacion</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Tiempo</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leditTiempoAct"/>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>93</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>106</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelarInfGeneral">
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnGuardarInfGeneral">
              <property name="text">
               <string>Aceptar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabConfRed">
        <attribute name="title">
         <string>Red</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Proxy</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Usuario</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leditPxUser"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Contraseña</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="leditPxPass">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="leditPxHost"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Puerto</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="leditPxPort">
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>106</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelarInfRed">
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnGuardarInfRed">
              <property name="text">
               <string>Aceptar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabConfBD">
        <attribute name="title">
         <string>Almacenamiento</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Base de Datos</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Nombre BD</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leditBDName"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Usuario</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="leditBDUser"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Contraseña</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="leditBDPass"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="leditBDHost"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Puerto</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="leditBDPort"/>
            </item>
           </layout>
           <zorder>label_5</zorder>
           <zorder>label_6</zorder>
           <zorder>label_7</zorder>
           <zorder>leditBDUser</zorder>
           <zorder>label_8</zorder>
           <zorder>leditBDPass</zorder>
           <zorder>leditBDHost</zorder>
           <zorder>leditBDPort</zorder>
           <zorder>label_9</zorder>
           <zorder>leditBDName</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>106</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelarInfBD">
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnGuardarInfBD">
              <property name="text">
               <string>Aceptar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>200</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConfigurar">
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Configure.png</normaloff>../Recursos/iconos/Configure.png</iconset>
   </property>
   <property name="text">
    <string>Configurar</string>
   </property>
   <property name="toolTip">
    <string>Configurar</string>
   </property>
  </action>
  <action name="actionDescargar">
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Downloads Library.png</normaloff>../Recursos/iconos/Downloads Library.png</iconset>
   </property>
   <property name="text">
    <string>Descargar</string>
   </property>
   <property name="toolTip">
    <string>Descargar</string>
   </property>
  </action>
  <action name="actionDetener">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Power - Shut Down.png</normaloff>../Recursos/iconos/Power - Shut Down.png</iconset>
   </property>
   <property name="text">
    <string>Detener</string>
   </property>
   <property name="toolTip">
    <string>Detener Actualizacion</string>
   </property>
  </action>
  <action name="actionActualizar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Power - Restart.png</normaloff>../Recursos/iconos/Power - Restart.png</iconset>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
   <property name="toolTip">
    <string>Actualizar</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Default.png</normaloff>../Recursos/iconos/Default.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="toolTip">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionCrear_Archivo">
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Notepad.png</normaloff>../Recursos/iconos/Notepad.png</iconset>
   </property>
   <property name="text">
    <string>Crear Archivo</string>
   </property>
   <property name="toolTip">
    <string>Crear Archivo</string>
   </property>
  </action>
  <action name="actionSalvar_Comp">
   <property name="icon">
    <iconset>
     <normaloff>../Recursos/iconos/Tasks.png</normaloff>../Recursos/iconos/Tasks.png</iconset>
   </property>
   <property name="text">
    <string>Salvar Comp.</string>
   </property>
   <property name="toolTip">
    <string>Salvar las competiciones que se van a mostrar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
